src/mlpack/methods/ann/augmented/tasks/add.hpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * @file add.hpp
+ * @author Konstantin Sidorov
+ *
+ * Definition of the AddTask class, which implements a generator of
+ * instances of sequence addition task.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+
+#ifndef MLPACK_METHODS_AUGMENTED_TASKS_ADD_HPP
+#define MLPACK_METHODS_AUGMENTED_TASKS_ADD_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace ann /* Artificial Neural Network */ {
+namespace augmented /* Augmented neural network */ {
+namespace tasks /* Task utilities for augmented */ {
+class AddTask
+{
+ public:
+  AddTask(size_t bitLen);
```

Comment:
Can you comment each parameter as we did for the other methods? Also, I think it would be good idea, to improve the general task description e.g. we could a simple example how to use the code and show an example output. What do you think?